processor/src/test/java/org/mapstruct/ap/testutil/runner/CompilingStatement.java

```diff
@@ -102,12 +102,7 @@ protected CompilationCache initialValue() {
         }
     };
 
-    private static final List<String> LIBRARIES = Arrays.asList(
```

Comment:
> Hmm, works for me. In what way does the test fail for you? Did you mvn clean install first?

It happens when running `mvn clean install`. The test expects a failed compilation due to the annotation type missing, but actually the compilation passes for me. I assume the annotation is made available to the test compilation by that changed approach for setting up the test class path.